jobbergate_api/config.py

```diff
@@ -0,0 +1,14 @@
+from prettyconf import config
```

Comment:
we should just use [pydantic.BaseSettings](https://pydantic-docs.helpmanual.io/usage/settings/) which comes with a ton of validation and is highly configurable, and is already in your dependency stack.